Source/Scripts/setup-upload-symbols.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Very well written script, good variable names, easy to follow, good logging. Thanks!